packages/theming/win32-theme/src/createPartialOfficeTheme.ts

```diff
@@ -0,0 +1,33 @@
+import { ColorRamp, OfficePalette, PartialTheme } from '@fluentui-react-native/theme-types';
+import { OfficeThemingModule } from './NativeModule/officeThemingModule';
+import { paletteFromOfficeColors } from './paletteFromOfficeColors';
+
+const createColorRamp = ({ values, index = -1 }: Partial<ColorRamp>) => ({
+  values,
+  index,
```

Comment:
I've seen `index` a few times, what does it get used for? And why do we default to -1?

Reply:
This was actually moved from theming-react-native, @FalseLobster might have better context for that.

Reply:
That was based on trying to align with a now out-of-date version of the Web's theming ideas.  I don't think it's relevant any more and should probably get removed since it's just complicating the theme dictionary.

Reply:
I agree, though I want the design we move towards to be a bit more stable before we transition.